Paper.tex

```diff
@@ -2030,6 +2045,8 @@ \section{Fee Schedule}\label{app:fees}
 \linkdest{G__selfdestruct}{}$G_{\mathrm{selfdestruct}}$ & 5000 & Amount of gas to pay for a {\small SELFDESTRUCT} operation. \\
 $G_{\mathrm{create}}$ & 32000 & Paid for a {\small CREATE} operation. \\
 \linkdest{G__codedeposit}{}$G_{\mathrm{codedeposit}}$ & 200 & Paid per byte for a {\small CREATE} operation to succeed in placing code into state. \\
+\linkdest{G__initcodeword}{}$G_{\mathrm{initcodeword}}$ & 2 & Paid per word of the initcode at the beginning of a deploy transaction, a {\small CREATE},\\
```

Comment:
[EIP-3860](https://eips.ethereum.org/EIPS/eip-3860#parameters) define it as its own `INITCODE_WORD_COST` constant, also I would find it confusing to use $G_{base}$ because of the description: "Amount of gas to pay for operations of the set $W_{base}$.". What do you think?

In another subject, your comment made me realize that function $N(x)$ was already used as pointed in your review of `EIP-3855` above, I'll change it to $R(x)$ which I think is not used.

Reply:
Changes in [#5a342a7](https://github.com/ethereum/yellowpaper/pull/906/commits/5a342a742d1c6c04b23b143a7c49b381b7a85660)

Reply:
> [EIP-3860](https://eips.ethereum.org/EIPS/eip-3860#parameters) define it as its own `INITCODE_WORD_COST` constant, also I would find it confusing to use Gbase because of the description: "Amount of gas to pay for operations of the set Wbase.". What do you think?
> 
> In another subject, your comment made me realize that function N(x) was already used as pointed in your review of `EIP-3855` above, I'll change it to R(x) which I think is not used.

Your call, don't have strong opinions, it can always be simplified post this PR.